scripts/create-conda-recipe.py

```diff
@@ -0,0 +1,188 @@
+import re
+from collections import defaultdict
+from copy import deepcopy
+
+import requests
+import yaml
+from bs4 import BeautifulSoup
+
+
+def get_extra_requires(path):
+    try:
+        with open(path) as fp:
+            extra_deps = defaultdict(set)
+            for k in fp:
+                if k.strip() and not k.startswith('#'):
+                    tags = set()
+                    if ':' in k:
+                        k, v = k.split(':')
```

Comment:
It might be better to use more descriptive variable names rather than letter variables such as `k` and `v` to improve readability?